tests/ttnn/unit_tests/operations/test_paged_update_cache.py

```diff
@@ -149,9 +201,9 @@ def check_zero(tensor):
 @pytest.mark.parametrize("max_seq_len", [2048])
 @pytest.mark.parametrize("num_users", [32])
 @pytest.mark.parametrize("num_heads", [1])
-@pytest.mark.parametrize("input_dtype", [ttnn.bfloat16, ttnn.bfloat8_b])
+@pytest.mark.parametrize("input_dtype", [ttnn.bfloat16])
 @pytest.mark.parametrize("cache_idx", [127, 1057])
-@pytest.mark.parametrize("cache_dtype", [ttnn.bfloat16, ttnn.bfloat8_b])
+@pytest.mark.parametrize("cache_dtype", [ttnn.bfloat8_b])
```

Comment:
Same comment about changing the test parameters. Should we keep the old parameters?